docs/ServiceNow-batchsink.md

```diff
@@ -23,6 +23,12 @@ Properties
 
 **Password**: The password for ServiceNow Instance.
 
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
+
+**Connection:** Name of the connection to use. Table Names information will be provided by the connection.
```

Comment:
Should this say Table Name not Names since I think the plugin writes to a single table?

Reply:
Done